data/sofa_demo.ipynb

```diff
@@ -0,0 +1,75 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sofa_utils\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tau_db_dir = '/scratch/ci411/SRIR_DATASETS/TAU_SRIR_DB/TAU-SRIR_DB'\n",
+    "sofa_db_dir = '/scratch/ci411/SRIR_DATASETS/TAU_SRIR_DB_SOFA'\n",
+    "db_name = \"TAU-SRIR-DB-SOFA\"\n",
+    "\n",
+    "for room_idx in range(9):\n",
+    "    for aud_fmt in ['foa','mic']:\n",
+    "        #load flattened (and flipped) rirs/paths from TAU-SRIR database\n",
+    "        rirs, source_pos, mic_pos, room = sofa_utils.load_flat_tau_srir(tau_db_dir, room_idx, aud_fmt=aud_fmt)\n",
+    "        filepath = os.path.join(sofa_db_dir, aud_fmt, room+'.sofa')\n",
+    "        comment = f\"SOFA conversion of {room} from TAU-SRIR-DB\"\n",
+    "        #create .sofa files with flattened rirs/paths + metadata\n",
+    "        sofa_utils.create_srir_sofa(filepath, rirs, source_pos, mic_pos, db_name=db_name,\\\n",
+    "                     room_name=room, listener_name=aud_fmt, sr=24000, comment=comment)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "load_rirs = sofa_utils.load_rir('sofa_file.sofa')\n",
+    "load_sp = sofa_utils.load_pos('sofa_file.sofa')"
```

Comment:
this loads the absolute coordinates, right? How can I get them normalized?

Reply:
[See here](https://github.com/iranroman/SELD-data-generator/pull/20/files#diff-1d64364bdf0922904a7de475e2e66ba59e16168e2741badd29bc90268e09f138R170), theres the kwarg `doas` which is `True` by default; you're already loading normalized coordinates! If you set `doas` to false, you'll get the aboslute coordinates.